.azure/wso2-opensearch-k8s-operator-build.yaml

```diff
@@ -68,3 +68,12 @@ steps:
     tags: |
       $(APP_REVISION)
       latest
+- script: |
+    cd charts/opensearch-operator
+    helm dependency build
+    echo "Packaging chart..."
+    helm package .
+    packageName=$(printf "%s-%s.tgz" "wso2-$(yq '.name' Chart.yaml)" $(yq '.version' Chart.yaml))
+    helm push $packageName oci://choreoprivateacr.azurecr.io/helm
+    helm push $packageName oci://choreopublicacr.azurecr.io/helm
```

Comment:
Does the docker login that happens from the above `Docker@2` task get used in this script? Or do I need to add a separate Docker login step before this script?
@SachithKasthuriarachchi

Reply:
you need to login to the ACR first similar to https://github.com/wso2-enterprise/choreo-private-dataplane-helm/blob/main/.azure/choreo-pdp-helm-release.yaml#L20-L29